applet/src/lastheard.c

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
currently lh already uses it's own buffer and logarea. my first try was using syslog, but didn't get it work asap, so I created a clone named lastheard. bringing both together may be a later goal for me, it needs some more review on syslog, to enhance this to handle different logbuffers.